website/static/js/share/results.js

```diff
@@ -105,14 +104,14 @@ var Description = {
         var showOnClick = function() {
             ctrl.showAll(!ctrl.showAll());
         };
-        if ((result.description || '').length > 350) {
+        if ((result.description || '').length > 350 && result.description.indexOf('$') === -1) {
             return m('', [
                 m('p.readable.pointer', {onclick: showOnClick},
                     ctrl.showAll() ? result.description : $.truncate(result.description, {length: 350})
                 ),
                 m('a.sr-only', {href: '#', onclick: showOnClick}, ctrl.showAll() ? 'See less' : 'See more')]);
         } else {
-            return m('p.readable', result.description);
+            return m('p.readable', MathJax.Hub.Queue(['Typeset', MathJax.Hub]), result.description);
```

Comment:
The build is failing because `MathJax` isn't explicitly defined at this point, as it gets included in the template. Consider including it here with `var MathJax = require('MathJax');`, and setting the `Hub.Config` in this file as well.